testsuite/tests/typing-modules/applicative_functor_type.ml

```diff
@@ -19,13 +19,17 @@ type t = Set.Make(M).t
 Line 1, characters 9-22:
 1 | type t = Set.Make(M).t
              ^^^^^^^^^^^^^
-Error: The type of M does not match Set.Make's parameter
-       Modules do not match:
-         sig type t = M.t val equal : 'a -> 'a -> bool end
-       is not included in
-         Set.OrderedType
-       The value `compare' is required but not provided
-       File "set.mli", line 52, characters 4-31: Expected declaration
+Error: The functor application Set.Make(M) is ill-typed.
+       These arguments:
+         M
+       do not match these parameters:
+         functor (Ord : Stdlib__set.OrderedType) -> ...
```

Comment:
Why is `Set` expanded here?

Reply:
This is a regression that I missed, `Stdlib__set` is not compressed to `Set` because it is not printed in the right environment. I will fix this regression.